src/randomizer.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const getDigit = () => Math.floor(Math.random() * 10);
+
+const get4Digits = () => {
+  const digits = [];
+
+  for (let i = 0; i < 4; i++) {
```

Comment:
I guess, it is possible to remake this logic without nested loops

Reply:
agree, I like your version way more